src/utils.ts

```diff
@@ -64,12 +64,25 @@ const sanitizeMessage = (message: string) => message.trim().replace(/[\n\r]/g, '
 
 const promptTemplate = 'Write an insightful but concise Git commit message in a complete sentence in present tense for the following diff without prefacing it with anything:';
 
+const getTranslatedPrompt = (lang: string) => {
+	const validLangs = ['en', 'es', 'jp', 'zh', 'de', 'fr', 'it', 'pt'];
```

Comment:
Where is this list from?

Reply:
I got that list by doing some research about the supported langs by GPT-3. I didn't find an official document about this, so I select the more popular languages where I tested, and GPT-3 works.

Also, I added that list to prevent users enter an inexisting language and avoid unexpected behaviors from GPT.

Reply:
In the future when there are requests to support more languages, we'll need a way to verify them.

Would you mind adding a comment that explains there's no official list and how you vetted them?

Reply:
You now what? I figured I could ask ChatPGT for that list. I did and updated the list